src/test/run-make/git_clone_sha1.sh

```diff
@@ -14,10 +14,10 @@ URL=$2
 SHA1=$3
```

Comment:
Github won't let me comment on lines that are unchanged, but please also change the shebang to `/usr/bin/env bash`.